ddp-workspace/projects/ddp-sdk/src/lib/services/configuration.service.ts

```diff
@@ -24,4 +24,9 @@ export class ConfigurationService {
     studyGuid: string;
     // country code if limiting app to just one country
     supportedCountry: string | null = null;
+    // whether dashboard status should display a count of questions
+    dashboardShowQuestionCount = false;
+    // if dashboardShowQuestionCount is true, exclude activity guids listed here from showing
+    // their question count
+    dashboardShowQuestionCountExceptions: string[] = [];
```

Comment:
Trying to keep this backward compatible and this is the minimal configuration I could think of to deliver requirements.